Frontend/ui-driver/src/Screens/NotificationsScreen/ScreenData.purs

```diff
@@ -16,7 +16,8 @@
 module Screens.NotificationsScreen.ScreenData where
 
 import Data.Maybe (Maybe(..))
-import MerchantConfig.DefaultConfig as DC
+import ConfigProvider
```

Comment:
Adding these two imports everytime are required? Can we create a ReExport if this is required? what's in Constants which is required here? 

Constants shouldn't be connected with configs ideally.

Reply:
Just to make the function parameterized. Instead of sending empty string or LazyCheck we are sending the key name itself. We can add a ReExport.

Reply:
from constants only key name is coming? can it be part of configprovider?

Reply:
Added Reexport for it. We can only import the config provider now.